examples/kbleds.c:64:55: error: ‘struct tty_driver’ has no member named ‘magic’

This error occurs on the new linux kernel v6.x :

```
lkmpg-git/examples$ make
make -C /lib/modules/6.2.0-26-generic/build CC=cc M=/data/source/linux/lkmpg/lkmpg-git/examples modules
make[1]: Entering directory '/usr/src/linux-headers-6.2.0-26-generic'
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: x86_64-linux-gnu-gcc-11 (Ubuntu 11.3.0-1ubuntu1~22.04.1) 11.3.0
  You are using:           cc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
  CC [M]  /data/source/linux/lkmpg/lkmpg-git/examples/kbleds.o
In file included from ./include/linux/kernel.h:29,
                 from ./arch/x86/include/asm/percpu.h:27,
                 from ./arch/x86/include/asm/nospec-branch.h:14,
                 from ./arch/x86/include/asm/paravirt_types.h:27,
                 from ./arch/x86/include/asm/ptrace.h:97,
                 from ./arch/x86/include/asm/math_emu.h:5,
                 from ./arch/x86/include/asm/processor.h:13,
                 from ./arch/x86/include/asm/timex.h:5,
                 from ./include/linux/timex.h:67,
                 from ./include/linux/time32.h:13,
                 from ./include/linux/time.h:60,
                 from ./include/linux/stat.h:19,
                 from ./include/linux/module.h:13,
                 from /data/source/linux/lkmpg/lkmpg-git/examples/kbleds.c:7:
/data/source/linux/lkmpg/lkmpg-git/examples/kbleds.c: In function ‘kbleds_init’:
/data/source/linux/lkmpg/lkmpg-git/examples/kbleds.c:64:55: error: ‘struct tty_driver’ has no member named ‘magic’
   64 |     pr_info("kbleds: tty driver magic %x\n", my_driver->magic);
      |                                                       ^~
./include/linux/printk.h:429:33: note: in definition of macro ‘printk_index_wrap’
  429 |                 _p_func(_fmt, ##__VA_ARGS__);                           \
      |                                 ^~~~~~~~~~~
./include/linux/printk.h:530:9: note: in expansion of macro ‘printk’
  530 |         printk(KERN_INFO pr_fmt(fmt), ##__VA_ARGS__)
      |         ^~~~~~
/data/source/linux/lkmpg/lkmpg-git/examples/kbleds.c:64:5: note: in expansion of macro ‘pr_info’
   64 |     pr_info("kbleds: tty driver magic %x\n", my_driver->magic);
      |     ^~~~~~~
make[2]: *** [scripts/Makefile.build:260: /data/source/linux/lkmpg/lkmpg-git/examples/kbleds.o] Error 1
make[1]: *** [Makefile:2026: /data/source/linux/lkmpg/lkmpg-git/examples] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-6.2.0-26-generic'
make: *** [Makefile:44: all] Error 2
```


Just comment this line `examples/kbleds.c:64` to workaround:

```
    //pr_info("kbleds: tty driver magic %x\n", my_driver->magic);
```

--- a/examples/kbleds.c
+++ b/examples/kbleds.c
@@ -61,7 +61,7 @@ static int __init kbleds_init(void)
     pr_info("kbleds: finished scanning consoles\n");
 
     my_driver = vc_cons[fg_console].d->port.tty->driver;
-    pr_info("kbleds: tty driver magic %x\n", my_driver->magic);
+    pr_info("kbleds: tty driver name %s\n", my_driver->driver_name);
 
     /* Set up the LED blink timer the first time. */
     timer_setup(&my_timer, my_timer_func, 0);

